src/lib.rs

```diff
@@ -428,6 +428,68 @@ const KSDATAFORMAT_SUBTYPE_PCM: [u8; 16] = [0x01, 0x00, 0x00, 0x00, 0x00, 0x00,
 const KSDATAFORMAT_SUBTYPE_IEEE_FLOAT: [u8; 16] = [0x03, 0x00, 0x00, 0x00, 0x00, 0x00, 0x10, 0x00,
                                                    0x80, 0x00, 0x00, 0xaa, 0x00, 0x38, 0x9b, 0x71];
 
+
+impl WavSpec {
+    /// Get "stand-alone" wav header representing infinite or unknown size wav file.
+    /// Use this if you need to write audio data to non-seekable sinks (like stdout).
+    ///
+    /// Actual samples are supposed to be written without `hound`'s help.
+    ///
+    /// Such wav files are produced e.g. by FFmpeg and have 0xFFFFFFFF instead of chunk sizes.
+    ///
+    /// Note that such files may be non-standard. Consider using `WavWriter` for better API.
+    ///
+    /// Example:
+    ///
+    /// ```no_run
+    /// extern crate hound;
+    /// use std::io::Write;
+    /// 
+    /// let spec = hound::WavSpec {
+    ///     bits_per_sample: 16,
+    ///     channels: 1,
+    ///     sample_format: hound::SampleFormat::Int,
+    ///     sample_rate: 16000,
+    /// };
+    /// 
+    /// let v = spec.into_header_for_infinite_file();
+    /// 
+    /// let so = std::io::stdout();
+    /// let mut so = so.lock();
+    /// so.write_all(&v[..]).unwrap();
+    /// 
+    /// loop {
```

Comment:
This would then become
```rust
use hound::Sample;
loop {
    for i in 0..1000_i16 {
        (i % 60).write(&mut so, spec.bits_per_sample).unwrap();
    }
}
```